src/code/z_effect_soft_sprite_old_init.c

```diff
@@ -745,12 +771,12 @@ void EffectSsFhgFlash_SpawnLightBall(PlayState* play, Vec3f* pos, Vec3f* velocit
 }
 
 /**
- * Spawn a shock effect
+ * Spawn a purple shock/lighting effect.
```

Comment:
think its probably best to just leave it at shock? dont think it resembles lightning at all really. Its just a bunch of "electricity" that doesnt really resemble much of a lightning strike

I know I put lightning in the description a couple years ago, but I think it was my mistake to do that lol

Reply:
Actually I wrote "lighting" here but I don't remember if that's a typo for "lightning" <img src="https://cdn.discordapp.com/emojis/690623121664901150.webp?size=56&quality=lossless" alt="Thonk" height="24">

Reply:
Probably a typo for "lightning", I guess I took that from `FhgFlashLightningParam` maybe or elsewhere

Reply:
lightning -> "a ball of electrical arcs" but I can also remove

Reply:
it doesnt add anything for me personally, but am fine with it staying